bigbone-rx/src/main/kotlin/social/bigbone/rx/admin/RxAdminIpBlocksMethods.kt

```diff
@@ -0,0 +1,94 @@
+package social.bigbone.rx.admin
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Single
+import social.bigbone.MastodonClient
+import social.bigbone.api.Pageable
+import social.bigbone.api.Range
+import social.bigbone.api.entity.admin.AdminIpBlock
+import social.bigbone.api.method.admin.AdminIpBlocksMethods
+
+/**
+ * Reactive implementation of [AdminIpBlocksMethods].
+ *
+ * Allow management of ip addresses blocked and to be blocked
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/ip_blocks/">Mastodon admin/ip_blocks API methods</a>
+ */
+class RxAdminIpBlocksMethods(client: MastodonClient) {
+
+    private val adminIpBlocksMethods = AdminIpBlocksMethods(client)
+
+    /**
+     * Show information about all blocked IP ranges.
+     * @param range optional Range for the pageable return value
+     * @see <a href="https://docs.joinmastodon.org/methods/ip_blocks/#get">Mastodon API documentation: methods/ip_blocks/#get</a>
+     */
+    @JvmOverloads
+    fun getAllIpBlocks(range: Range = Range()): Single<Pageable<AdminIpBlock>> {
+        return Single.fromCallable {
+            adminIpBlocksMethods.getAllIpBlocks(range = range).execute()
+        }
+    }
```

Comment:
```suggestion
    fun getAllIpBlocks(range: Range = Range()): Single<Pageable<AdminIpBlock>> = Single.fromCallable {
        adminIpBlocksMethods.getAllIpBlocks(range = range).execute()
    }
```

Could you please use expression bodies here and for the other methods instead? 😊